### ISSUE TITLE
Patch / build / map version detection

### ISSUE THREAD

**Issue by timkurvers** [timkurvers/redota#24]
> ReDota currently uses the 7.23 map as a background.
> Games played on patch 7.29 may consequently look odd.

Optimally, ReDota should ship with maps and minimaps for the most recent patches.

A patch list is [included in dotaconstants](https://github.com/odota/dotaconstants/blob/master/json/patch.json) but may not be sufficient to handle patches for both local replays and Valve matches.

**Comment by likwidoxigen** [comment]
Greetings!
I've been self-hosting a 7.29 map and then using inspect to swap images and it's been pretty successful. 
Do you have an idea for how you'd like multi-maps implemented? Are you open to pull requests if I get something going?

**Comment by timkurvers** [comment]
Hil, glad it's been working out nicely!

ReDota currently attempts to [retrieve the build number from the game directory](https://github.com/timkurvers/redota/blob/master/src/lib/parser/Parser.js#L299-L305), resulting in something like: `4489`.

Unfortunately, that build number does not exist for replays locally recorded using the `record`-command and I have yet to find a list of these build numbers somewhere so that we can map them to the correct patches.

So, alternatively, I think it might be easier to use the `endedAt`-timestamp for a match, which is guaranteed to be set and can also be used directly to look up the 'correct' patch in https://github.com/odota/dotaconstants/blob/master/json/patch.json.

The images could be placed right next to the current 7.23 versions:

```
public/images/
├── map
│   └── 7.23.webp
└── minimap
    └── 7.23.webp
```

Pull requests more than welcome. 👍

**Comment by timkurvers** [comment]
Map and minimap resources for 7.29 were added in #48.

Keeping this issue open as version-detection and usage of the appropritate resources still remains to be implemented.

**Comment by timkurvers** [comment]
Fully implemented in 3941beca298571329b3aeeaa693a308fbbd52b5f with support for 7.23 and the 7.33 map.
